src/Categories/Category/BinaryProducts.agda

```diff
@@ -94,6 +94,12 @@ record BinaryProducts : Set (levelOfTerm 𝒞) where
   ⁂-cong₂ : f ≈ g → h ≈ i → f ⁂ h ≈ g ⁂ i
   ⁂-cong₂ = [ product ⇒ product ]×-cong₂
 
+  first-cong : f ≈ g → f ⁂ h ≈ g ⁂ h
```

Comment:
For consistency, I think these ought to be called `⁂-cong` with l/r superscripts.

Reply:
I called them that because we do define `first` and `second` rather than `⁂ˡ` and `⁂ʳ` or anything. I'm not sure why I didn't use those in the type signature

Reply:
Good point, I didn't remember that (nor didn't see them in the diffs to remind me). Right, no need to change them.